examples/react-whiteboard/package.json

```
@@ -9,8 +9,8 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "@liveblocks/client": "^0.17.11",
-    "@liveblocks/react": "^0.17.11",
+    "@liveblocks/client": "^0.18.0",
+    "@liveblocks/react": "^0.18.0",
```

You may want to bump this to 0.18.1 while you're at it, and also rerun `npm i` to update the lock file.

```suggestion
    "@liveblocks/client": "^0.18.1",
    "@liveblocks/react": "^0.18.1",
```